tests/security/audit/auditctl.pm

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Controlling the Audit system using auditctl
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772 tc#1768551
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $audit_rules = '/etc/audit/rules.d/audit.rules';
+    select_console 'root-console';
+
+    # Disable audit
+    validate_script_output('auditctl -e 0', sub { m/enabled 0/ });
+
+    # Check audit status
+    validate_script_output('auditctl -s', sub { m/enabled 0/ });
+
+    # Enable audit
+    validate_script_output('auditctl -e 1', sub { m/enabled 1/ });
+
+    # Check audit status again
+    validate_script_output('auditctl -s', sub { m/enabled 1/ });
+
+    # List all rules by default
+    validate_script_output('auditctl -l', sub { m/-a never,task/ });
```

Comment:
Shawn,

IMO, you should add more checking here. e.g.

1. after disabling the auditd, you will not able to get any auditd logs/record
2. after enabling the auditd, you are able to get auditd logs/record.
Any way,  I think cc tests have covered this part, you can double check that. 

You only check the command return value, but no function check, I would suggest you add it

Reply:
Good point!
I'm gonna check the function by:
1. stop auditd and clear audit.log
2. start some service (e.g. apparmor)
3. check audit.log, should be empty
4. start auditd and clear audit.log, repeat steps above